.template_system_tests.yml

```diff
@@ -0,0 +1,24 @@
+parameters:
```

Comment:
how about 1-line comment
```
#
# Run system tests for the specified module on the specified platform.
#
```

Reply:
and/or can we give this file any name? something that provide more context like `run_module_system_tests.yml`